internal/config/config.go

```diff
@@ -83,11 +82,15 @@ var Runtime *Config
 
 func init() {
 	// Set user config directory.
+	// notest
 	d, err := os.UserConfigDir()
-	errpkg.CheckFatal(err, "Unable to get the user config directory.")
+	if err != nil {
+		Logger.Fatal("Could not get retrieve Juno config directory.")
```

Comment:
Typo. "Could not ~~get~~ retrieve Juno config directory.". I'd also remove the word "Juno" as well.